README.md

```diff
@@ -374,3 +379,75 @@ $ git add git_deploy_key.enc .travis.yml
 $ git commit -m "ci(travis): Add the encrypted private ssh key"
 $ git push
 ```
+
+##### Add the SSH private key to Circle CI
+
+First we encrypt the `git_deploy_key` (private key) using a symmetric encryption (AES-256).  Run the folllowing `openssl` command and *make sure to note the output which we'll need later*:
+
+```bash
+$ openssl aes-256-cbc -e -p -in git_deploy_key -out git_deploy_key.enc -K `openssl rand -hex 32` -iv `openssl rand -hex 16`
+salt=711624E085AB48A4
```

Comment:
Maybe replace the actual key/salt/iv with something like XXXXXXXX, YYYYYYYY, ZZZZZZZ etc...
I saw many users getting confused and copying/pasting the key from the example, creating unknowingly a bit security risk.

Reply:
Oh wow, thanks for this input, users were never my forte :)  I went for SSSSSSSS, KKKKKKK, VVVVV which I hope was super clear, see what you think?